src/components/_modal/pages/material/Material.module.css

```diff
@@ -71,3 +71,8 @@
   margin-left: var(--pt2);
   width: 50%;
 }
+
+/* Specificity to override */
+button.buttonDelete[data-cy="order-button"] {
+  white-space: nowrap;
+}
```

Comment:
Jeg ville nok finde en anden data-attribute end data-cy, hvis jeg var dig. Det bliver lidt for sammenkoblet hvis du spørger mig